packages/livechat/src/lib/uiKit.js

```diff
@@ -104,16 +104,22 @@ export const triggerAction = async ({ appId, type, actionId, rid, mid, viewId, c
 
 	try {
 		const result = await Promise.race([
-			Livechat.rest.post(`/apps/ui.interaction/${appId}`, {
-				type,
-				actionId,
-				rid,
-				mid,
-				viewId,
-				container,
-				triggerId,
-				payload,
-			}),
+			Livechat.sendUiInteraction(
+				{
+					type,
+					actionId,
+					rid,
+					mid,
+					viewId,
+					container,
+					triggerId,
+					payload,
+				},
+				{
+					appId,
+					visitorToken: Livechat.credentials.token,
```

Comment:
what is the advantage of having a method in the sdk that cannot even pass the automatic token to you?